chain/jsonrpc/src/lib.rs

```diff
@@ -698,7 +696,7 @@ impl JsonRpcHandler {
         near_jsonrpc_primitives::types::transactions::RpcTransactionError,
     > {
         if request_data.wait_until == TxExecutionStatus::None {
-            self.send_tx_async(request_data).await;
+            self.send_tx_async(request_data);
```

Comment:
same, why no await? I get that the result is not used but doesn't it change the timing of things?